terraform/acl_rules.tf

```diff
@@ -2,24 +2,24 @@
 # Restrictions by source IP are handled by the security group rules
 resource "aws_network_acl_rule" "ingress_from_anywhere_via_any_port" {
   network_acl_id = "${aws_network_acl.vuln_acl.id}"
-  egress = false
-  protocol = "-1"
-  rule_number = 100
-  rule_action = "allow"
-  cidr_block = "0.0.0.0/0"
-  from_port = 0
-  to_port = 0
+  egress         = false
```

Comment:
I detest this style, but Terraform does appear to advocate it.

Reply:
I don't love it, but since `terraform_fmt` takes care of it automatically, I don't really care that much.